python/lsst/pipe/tasks/ingest.py

```diff
@@ -17,6 +17,7 @@
 import lsst.pex.exceptions
 from lsst.pipe.base import Task, InputOnlyArgumentParser
 import lsst.afw.image as afwImage
+from lsst.afw.fits import DEFAULT_HDU
```

Comment:
OK, but I really don't like these imports of specific symbols from packages.  Not worth changing, but I'd much rather see `import lsst.afw.fits as afwFits` ...  `afwFits.DEFAULT_HDU`